daemon/container_operations_unix.go

```diff
@@ -718,6 +722,11 @@ func (daemon *Daemon) DisconnectFromNetwork(container *container.Container, n li
 	if err := container.ToDiskLocking(); err != nil {
 		return fmt.Errorf("Error saving container to disk: %v", err)
 	}
+
+	attributes := map[string]string{
+		"container": container.ID,
+	}
+	daemon.LogNetworkEventWithAttributes(n, "disconnect", attributes)
 	return nil
 }
 
```

Comment:
In this file can you please also take care of
`func (daemon *Daemon) releaseNetwork(container *Container)`

Given it is for a forced sandbox removal, I think we should generate the disconnect event from each connected network with something like:
```
+	attributes := map[string]string{
+		"container": container.ID,
+	}
+
 	for n := range networks {
+		if nw, err := daemon.FindNetwork(n); err == nil {
+                     daemon.LogNetworkEventWithAttributes(n, "disconnect", attributes)
+		}
 		networks[n] = &network.EndpointSettings{}
 	}
```